src/south/RestApi/RestApi.class.js

```diff
@@ -0,0 +1,178 @@
+const fs = require('fs')
+const path = require('path')
+const csv = require('papaparse')
+const moment = require('moment-timezone')
+const ProtocolHandler = require('../ProtocolHandler.class')
+
+/**
+ * Class RestApi
+ */
+class RestApi extends ProtocolHandler {
+  /**
+   * Constructor for RestApi
+   * @constructor
+   * @param {Object} dataSource - The data source
+   * @param {Engine} engine - The engine
+   * @return {void}
+   */
+  constructor(dataSource, engine) {
+    super(dataSource, engine)
+
+    const {
+      apiType,
+      host,
+      port,
+      entity,
+      authentication,
+      connectionTimeout,
+      requestTimeout,
+      startDate,
+    } = this.dataSource.RestApi
+
+    this.apiType = apiType
+    this.host = host
+    this.port = port
+    this.entity = entity
+    this.authentication = authentication
+    this.connectionTimeout = connectionTimeout
+    this.requestTimeout = requestTimeout
+    this.startDate = startDate // "startDate" is currently a "hidden" parameter of oibus.json
+
+    const { engineConfig: { caching: { cacheFolder } } } = this.engine.configService.getConfig()
```

Comment:
41 -> 47. Maybe this code should be generic and push to the protocolHandler class 
this.tmpFolder = this.getTmpFolder()

Reply:
Gonna do this in another PR with bulk merged.